core/services/vrf/vrf_coordinator_solidity_crosscheck_test.go

```diff
@@ -36,6 +35,8 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/internal/cltest"
 )
 
+const DefaultGasLimit uint32 = 500000
```

Comment:
Same here. Nbd since they are tests, but a bit easier for readers to understand the scope.